sendKindle.py

```diff
@@ -173,7 +173,7 @@ def send_mail(self):
                 mail_server = smtplib.SMTP(host=self.smtp_server, port=self.smtp_port)
                 mail_server.starttls()
             else:
-                mail_server = smtplib.SMTP_SSL(host=self.smtp_server, port=self.smtp_port)
+                mail_server = smtplib.SMTP(host=self.smtp_server, port=self.smtp_port)
```

Comment:
No. Instead of disabling SSL for everyone, add a new command line option `--no-ssl` that will populate `self.ssl` value accordingly. This will allow users to disable SSL if they wish. Use it in the `if` clause above. Instead of `if self.smtp_port == 587:` use something like `if not self.ssl:`.

Reply:
You are right. I may add a --no-ssl option later and create another pull request.
Thank you @kparal  :-)

Reply:
It's much better to update this pull request instead of creating a new one. Simply adjust the code in your branch and this PR will get automatically updated.